cloud/docker_client.go

```diff
@@ -104,6 +105,11 @@ func (c *dockerClientImpl) Init(apiVersion string) error {
 // EnsureImageDownloaded checks if the image in s3 specified by the URL already exists,
 // and if not, creates a new image from the remote tarball.
 func (c *dockerClientImpl) EnsureImageDownloaded(ctx context.Context, h *host.Host, url string) (string, error) {
+	const (
+		maxImageInspectAttempts = 40
+		retryInterval           = 15 * time.Second
```

Comment:
what if this gets aborted in the middle? we tend to get impatient and kill machines if they take more than 30 seconds to bounce.